### PR TITLE
feat: Add a filter by assignees

### PR DESCRIPTION
**Pull request by peterpaints**
#### What does this PR do?
- Adds a dropdown with a list of assignees, so that incidents can be filtered by assignee.
- This only works for admins.
#### Description of Task to be completed?
Show a dropdown with a list of assignees, then incidents can be filtered by assignee.
Note: The above feature should only be visible to the admin
 - Assignees who are not admins can only see their own assigned incidents
 - Assignees cannot assign incidents(only admins can), but they can CC.
#### Where should the reviewer start?
View file changes on PR.
#### How should this be manually tested?
- Log in to wire on localhost as an admin. Assign or CC a few test users on a couple of incidents. You can now filter incidents by those assigned to a specific user. 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#160442794](https://www.pivotaltracker.com/story/show/160442794)
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,4 @@
-NODE_ENV=staging
-API_URL=https://api-staging-wire.andela.com
+NODE_ENV=development
+API_URL=http://localhost:3000/api
```
Comment (kimobrian) [inline .env]: @jimmykimani Take over this task. Revert the changes to the .env file and ensure that's it working as expected. It was meant to add a filter by assignees

Comment (jimmykimani) [reply .env]: Noted

**Comment by jimmykimani** [comment]
@kimobrian made the changes working as expected! adding a filter by assignees
